.github/workflows/update.yml

```diff
@@ -0,0 +1,52 @@
+name: Update flake inputs
+on:
+  schedule:
+    # Update every Sunday and Wednesday
+    - cron: "51 3 * * 0,3"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+        persist-credentials: false
+        fetch-depth: 0
```

Comment:
Perhaps these lines should be indented one more step?

Reply:
Good catch, thanks.